1598607168594	Sync.LogManager	DEBUG	Flushing file log
1598607168608	Sync.LogManager	DEBUG	Log cleanup threshold time: 1597743168607
1598607168621	Sync.LogManager	DEBUG	Done deleting files.
1598607525253	Sync.Service	DEBUG	User-Agent: Firefox/79.0 (Linux x86_64) FxSync/1.81.0.20200720193547.desktop
1598607525253	Sync.Service	INFO	Starting sync at 2020-08-28 09:38:45 in browser session ox_tU9hUXzQf
1598607525254	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1598607525256	Sync.Status	INFO	Resetting Status.
1598607525256	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1598607525257	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1598607525257	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1598607525257	Sync.ErrorHandler	ERROR	Sync encountered an error: Error: Can't sync: Network is offline(resource://services-sync/stages/enginesync.js:59:13) JS Stack trace: sync@enginesync.js:59:13
onNotify@service.js:1356:28
WrappedNotify@util.js:210:32
WrappedLock@util.js:166:27
_lockedSync@service.js:1375:6
sync/<@service.js:1339:18
WrappedCatch@util.js:133:27
sync@service.js:1340:7
1598607525258	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1598607525258	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 4286324 ms.
1598607525259	Sync.SyncScheduler	DEBUG	Next sync in 4286324 ms. (why=client-backoff-schedule)
1598607525262	Sync.Service	DEBUG	Exception calling anonymous function: Error: Can't sync: Network is offline(resource://services-sync/stages/enginesync.js:59:13) JS Stack trace: sync@enginesync.js:59:13
onNotify@service.js:1356:28
WrappedNotify@util.js:210:32
WrappedLock@util.js:166:27
_lockedSync@service.js:1375:6
sync/<@service.js:1339:18
WrappedCatch@util.js:133:27
sync@service.js:1340:7
